src/app/results/results.component.ts

```diff
@@ -95,7 +113,7 @@ export class ResultsComponent implements OnInit {
   getStyle(page) {
     return ((this.presentPage) === page);
   }
-  constructor(private searchservice: SearchService, private route: Router, private activatedroute: ActivatedRoute,
+  constructor(private domsanitizer: DomSanitizer, private searchservice: SearchService, private route: Router, private activatedroute: ActivatedRoute,
```

Comment:
maximum line length should be 140 characters. maybe you could break up the line a bit to avoid the lint error